README.md

```diff
@@ -1,2 +1,30 @@
 # activemq-integration
```

Comment:
there are 16 issues with this readme.  Run mdl --warnings on your .md files before making a PR.